lib/excision_web/controllers/classifier_controller.ex

```diff
@@ -67,4 +67,160 @@ defmodule ExcisionWeb.ClassifierController do
       send_resp(conn, :no_content, "")
     end
   end
+
+  operation :invoke,
+    summary: "Invoke classifier",
+    description: "Invoke a classifier"
+
+  def invoke(conn, %{"id" => id}) do
+    classifier = Excisions.get_classifier!(id)
+
+    decision_site =
+      Excisions.get_decision_site!(classifier.decision_site_id, preloads: [:choices])
+
+    if Excisions.is_default_passthrough_classifier?(classifier) do
+      proxy_openai_structured_response(conn, classifier, decision_site)
+    else
+      # TODO: this is really slow, need GenServer (Agent?) to keep model in memory
+      # TODO: read model name from classifier
+      model_name = classifier.base_model_name
+
+      {:ok, tokenizer} = Bumblebee.load_tokenizer({:hf, model_name})
```

Comment:
Please use the Util.make bumble bee repo so that we ensure this uses creds etc

Reply:
That's a good idea :)

Reply:
Let's defer to https://linear.app/blueteam/issue/BT-891/use-utilmake-bumblebee-repo-to-ensure-hf-credsd